tests/datasource/fluent/test_metadatasource.py

```diff
@@ -528,7 +528,6 @@ def test_update_datasource_with_datasource_object(
                     "csv_asset": {
                         "batching_regex": "(?P<file_name>.*).csv",
                         "name": "csv_asset",
-                        "order_by": [],
```

Comment:
This was present because this value was being set in the manually defined `add_csv_asset()` method.